components/sysutils/tcpdump/Makefile

```diff
@@ -40,18 +41,13 @@ COMPONENT_LICENSE_FILE=	LICENSE
 # Prevent macro redefinition error 
 CFLAGS += -D _NETINET_IP_H
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/common.mk
 
 # fix my alignment issues instead of falling over on sparc
 studio_ALIGN.sparc.64=-xmemalign=16i
 CONFIGURE_OPTIONS.sparc += ac_cv___attribute__=no
 CONFIGURE_OPTIONS += $(CONFIGURE_OPTIONS.$(MACH))
 
-# Install binaries to /usr/sbin
-CONFIGURE_SBINDIR.64 = $(CONFIGURE_SBINDIR.32)
-
```

Comment:
Why do you change the location from sbin to bin?

Reply:
This Configure option doesn't have any effect on the updated version.
If it is set, the binaries are installed to /usr/bin anyway.